aks/redis/resources.tf

```diff
@@ -22,6 +22,7 @@ resource "azurerm_redis_cache" "main" {
   minimum_tls_version           = var.azure_minimum_tls_version
   public_network_access_enabled = var.azure_public_network_access_enabled
   redis_version                 = var.server_version
+  zones                         = var.azure_sku_name != "Premium" ? null : ["1", "2"]
```

Comment:
As discussed, redis multi AZ is not only Premium, as per https://azure.microsoft.com/en-gb/pricing/details/cache/?cdn=disable
We suggested the logic should be "NOT Basic and NOT Standard"

Reply:
updated condition